plugins/cue_points/base/lib/kCuePointManager.php

```diff
@@ -30,6 +32,19 @@ public function shouldConsumeDeletedEvent(BaseObject $object)
 	}
 	
 	/* (non-PHPdoc)
+	 * @see kObjectCreatedEventConsumer::shouldConsumeCreatedEvent()
+	*/
+	public function shouldConsumeCreatedEvent(BaseObject $object)
+	{
+		if($object instanceof mediaInfo)
```

Comment:
OK, since shouldConsume*() methods are there in order not to produce logs, I will shift the check to shouldConsumeCreatedEvent().